plat/kvm/x86/entry64.S

```diff
@@ -40,24 +40,24 @@
 .section .data.boot
 
 .align 8
-multiboot_header:
+_multiboot_header:
         /*  magic */
         .long   MULTIBOOT2_HEADER_MAGIC
         /*  ISA: i386 */
         .long   MULTIBOOT_ARCHITECTURE_I386
         /*  Header length. */
-        .long   multiboot_header_end - multiboot_header
+        .long   multiboot_header_end - _multiboot_header
         /*  checksum */
-        .long   -(MULTIBOOT2_HEADER_MAGIC + MULTIBOOT_ARCHITECTURE_I386 + (multiboot_header_end - multiboot_header))
+        .long   -(MULTIBOOT2_HEADER_MAGIC + MULTIBOOT_ARCHITECTURE_I386 + (multiboot_header_end - _multiboot_header))
 
 address_tag_start:
         .short MULTIBOOT_HEADER_TAG_ADDRESS
         .short MULTIBOOT_HEADER_TAG_OPTIONAL
         .long address_tag_end - address_tag_start
         /*  header_addr */
-        .long   multiboot_header
+        .long   _multiboot_header
         /*  load_addr */
-        .long   _start
+        .long   _libkvmplat_start32
```

Comment:
From the [multiboot documentation](https://www.gnu.org/software/grub/manual/multiboot2/multiboot.html), section 3.1.5 this address should be start of the text segment. This should either point to _text instead of _libkvmplat_start32

Reply:
I missed that, oops